### PR TITLE
Adding VEBA Evolutions to the website

### LINKED ISSUE #483: Add a release timeline (shortcode) to the VEBA website
**Describe the solution you'd like**
It would be great to have a timeline feature on the VEBA website implemented, which highlight e.g. **dates**, **releases** as well as the shipped **updates** which were introduced with such.

Example from my current VMUG slides:

veba_release_timeline

Example `shortcode`: [Hugo Timeline Shortcode - MetalBlueberry](https://metalblueberry.github.io/post/howto/2021-02-28_hugo_timeline_shortcode/).

Another great example of such an implementation is available on @pksrc blog [HERE](https://cloudtekki.com/post/looking-back-at-2020/) <-- .

### PR DESCRIPTION
**Pull request by pksrc**
## Summary

Please describe your changes and what you plan to introduce as part of this pull request submission. If it fixes a bug or resolves a feature request, be sure to link to the issue numbers following the template below.

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [x] Please ensure that you are making a pull request against the **Development** branch
- [x] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [x] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [x] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commits related to your change
- [x] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [x] Please include any relevant screenshots and/or output as part of your testing
- [x] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation changes
- [ ] Other (please describe):

## Resolved Issues

List of Issues closed or resolved by this PR. Add multiple `Closes` keyword followed by the issue number (e.g. Closes #ISSUE-NUMBER)

Closes #483 

## Additional Information

* Any other details you wish to include or mention

If you have any questions/comments, feel free to reach out to team on Slack [#vcenter-event-broker-appliance](https://vmwarecode.slack.com/archives/CQLT9B5AA)

Thank you from the VEBA Team! 🥳

### CODE REVIEW

**Comment by embano1** [comment]
AWESOME WORK!

File: docs/site/evolutions.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+- title: Launching VEBA 0.1.0
+  type: feature
+  date: Nov 2019
+  id: vzeroone
+  description: VEBA Fling v0.1.0 released live at VMworld Europe - 'Do it, do it, do it\' chants when @embano1 and @lamw ask if they should release the vCenter Event Broker Appliance fling (powered by @openfaas) live during their [session!](https://twitter.com/bbrundert/status/1192366254570508288)
+  detail: 
+    subtitle: Features
+    text: 
+    - Introduced VEBA v0.1.0 with OpenFaaS as the default Event Processor​ and vCenter as the Event Provided
```
Comment (lamw) [inline docs/site/evolutions.md:22]:
```suggestion
    - Introduced VEBA v0.1.0 with OpenFaaS as the default Event Processor​ and vCenter as the Event Provider
```

File: docs/site/evolutions.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  
+- title: VEBA v0.3.0
+  type: feature
+  date: Mar 2020
+  id: vzerothree
+  detail: 
+    subtitle: Features
+    text: 
+    - Introduced VMware Event Router
+    - Added support for AWS EventBridge as the Event Processor
```
Comment (lamw) [inline docs/site/evolutions.md:42]:
```suggestion
    - Added support for AWS EventBridge as an Event Processor
```

File: docs/site/evolutions.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+- title: Listed in CloudEvents.io
+  type: milestone
+  date: Oct 2020
+  id: cloudevents
+  description: VEBA listed on CloudEventsIO site! - [details](https://twitter.com/lamw/status/1362572308754305024)
+
+- title: VEBA v0.6.0
+  type: feature
+  date: Apr 2021
+  id: vzerofive
```
Comment (lamw) [inline docs/site/evolutions.md:87]:
```suggestion
  id: vzerosix
```

File: docs/site/evolutions.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  detail: 
+    subtitle: Features
+    text: 
+    - Embedded Knative (new default Event Processor)​
+    - vSphere UI integration (h5 Client Plugin)​
```
Comment (lamw) [inline docs/site/evolutions.md:92]:
```suggestion
    - vSphere UI integration (H5 Client Plugin)​
```

**Review by lamw — CHANGES_REQUESTED** [review]
Couple of minor changes.

I noticed in the screenshot for items located on the right, the text is aligned `right`, would it read better to have text aligned `left` next to bullets?

**Comment by pksrc** [comment]
preview of the latest changes as requested
image

**Review by lamw — APPROVED** [review]
🥳 

Awesome work @pksrc

**Pull request merged**